### DISCUSSION TITLE
Cyberduck redesign for Windows - Download files

### DISCUSSION THREAD

**Discussion by janwaelti98** [iterate-ch/cyberduck#14512] [Polls]
Dear Cyberduck Community,

We want to improve the user experience of Cyberduck on Windows 11 and update the user interface to meet Microsoft's current design guidelines. To ensure that we meet the needs and expectations of our community, we would like to involve you in the design process.

That's why we invite you to participate in our survey. We want to give you the opportunity to provide feedback on various UI elements to ensure Cyberduck is as user-friendly as possible. This first survey focuses on the basic layout of Cyberduck. We are also working on a new tab concept to better manage multiple connections. We have a few questions about this as well.

Your participation in this survey is essential for us to ensure that we can provide the best possible user experience. We would like to thank you in advance and look forward to your feedback.

Best regards,
The Cyberduck Team

<img width="856" alt="Screenshot 2023-04-11 at 16 03 01" src="https://user-images.githubusercontent.com/79445035/231188011-03a197c1-4943-44ee-b850-7f831fa8a214.png">

**Comment by dkocher** [comment]
Another important often used option is to drag files to Windows Explorer.

**Reply by janwaelti98** [reply]
Thank you! I completely forgot about that.

**Reply by enisio** [reply]
You cannot design the user interface exactly the same as the mac. For example, add the Search section in the Transfer window (as in MAC). Biggest need for me.
![Screenshot_1](https://github.com/iterate-ch/cyberduck/assets/28567481/10440406-d5cb-4b14-86f6-4695bc1cfebb)
